hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestPermissionCheck.java

```diff
@@ -93,14 +95,8 @@ public void testListS3Buckets() throws IOException {
     doThrow(exception).when(objectStore).getS3Volume();
     RootEndpoint rootEndpoint = new RootEndpoint();
     rootEndpoint.setClient(client);
-
-    try {
-      rootEndpoint.get();
-      Assert.fail("Should fail");
-    } catch (Exception e) {
-      Assert.assertTrue(e instanceof OS3Exception);
-      Assert.assertTrue(((OS3Exception) e).getHttpCode() == HTTP_FORBIDDEN);
-    }
+    OS3Exception e = assertThrows(OS3Exception.class, () -> rootEndpoint.get());
+    assertEquals(e.getHttpCode(), HTTP_FORBIDDEN);
```

Comment:
nit: expected is supposed to be on left side and actual on the right.

Reply:
Done, Thanks,